File: src/MaksIT.WebUI/src/models/letsEncryptServer/account/requests/PatchAccountRequest.ts
import { PatchAction } from '@/models/PatchAction'
import { PatchHostnameRequest } from './PatchHostnameRequest'

export interface PatchAccountRequest {
  description?: PatchAction<string>
  isDisabled?: PatchAction<boolean>
  contacts?: PatchAction<string>[]
  hostnames?: PatchHostnameRequest[]
}

File: src/MaksIT.CertsUI/Models/CertsUI/Account/Requests/PatchAccountRequest.cs
using MaksIT.Core.Abstractions.Webapi;


namespace MaksIT.CertsUI.Models.CertsUI.Account.Requests;

public class PatchAccountRequest : PatchRequestModelBase {
  public string? Description { get; set; }
  public bool? IsDisabled { get; set; }
  public List<string>? Contacts { get; set; }
  public List<PatchHostnameRequest>? Hostnames { get; set; }
}
